samples/BuilderApp/Program.cs

```diff
@@ -101,6 +101,8 @@ private static void Usage()
         }
     }
 
+    // Class for performing the project build
+    // The Microsoft.Build dlls needed from within aren't loaded until after the class is instantiated
```

Comment:
Also, a minor nit: the "C#-y" way to comment this would be in the form of an [XML doc comment](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc), something like

```c#
/// <summary>
/// Class for performing the project build
/// </summary>
/// <remarks>
/// This is in a separate class because ...
/// </remarks>
```